support-frontend/assets/pages/[countryGroupId]/events/events.tsx

```diff
@@ -64,7 +62,12 @@ export function Events() {
 			header={<Header />}
 			footer={
 				<FooterWithContents>
-					<ThankYouFooter />
+					<ThankYouFooter
+						copyOverride="&nbsp;"
+						cssOverrides={css`
+							margin-bottom: ${space[3]}px;
+						`}
+					/>
```

Comment:
Should we just remove this from the events page and create a new element here?

Reply:
Agree, too complicated, have removed it completely as you suggest, thanks.